go_to_office/lib/screens/admin/seats_list.dart

```diff
@@ -1,8 +1,9 @@
 import 'package:flutter/material.dart';
-
-import '../../model/seat.dart';
-import '../../util/strings.dart';
 import 'seat.dart';
+import '../../util/strings.dart';
+import '../../model/seat.dart';
+import '../../util/repository.dart';
+
 
 final List<Seat> SeatsList = [
   Seat(location: 'Herzeliya', id: '1', roomId: '1'),
```

Comment:
Maybe it's a good time to change this initial value?

Reply:
Since we are not getting the seats yet from the DB - I'd rather leave a mock seat

Reply:
@Orlyga01